.github/vale/dicts/aiven.dic

```diff
@@ -126,6 +128,7 @@ Singlestat
 Snowflake
 Splunk
 spring
+sstableloader
```

Comment:
I believe we are trying to avoid adding tool names in the vocab. I suggested few changes of tool_name to `tool_name` that should make this line not needed